spec/helpers/application_helper/toolbar_builder_spec.rb

```diff
@@ -1727,6 +1727,17 @@ def setup_firefox_with_linux
       end
     end
 
+    it 'disables the add new iso datastore button when no EMSes are available' do
+      allow(ManageIQ::Providers::Redhat::InfraManager)
```

Comment:
Do we need the `allow` :hammer: , can it be an `expect` instead?  We know the method should be called.